### DISCUSSION TITLE
CAN‘T USE fastddsgen：Error: Unable to access jarfile

### DISCUSSION THREAD

**Discussion by stronghasaki** [eProsima/Fast-DDS-Gen#283] [Q&A]
![图片](https://github.com/eProsima/Fast-DDS-Gen/assets/105260958/5491eda4-8e11-4648-a7c2-1604799a8b38)
Error: Unable to access jarfile

**Comment by JesusPoderoso** [comment]
Hi @stronghasaki, thanks for using Fast DDS.
Please, make sure you have [compiled Fast DDS-Gen](https://fast-dds.docs.eprosima.com/en/latest/installation/sources/sources_linux.html#compiling-fast-dds-gen).
Try to run `./gradlew assemble` to generate the .jar file.
